statvar_imports/ndap/nss_health/pvmap/metadata_seven.csv

```diff
@@ -0,0 +1,17 @@
+parameter,value

```

Comment:
The metadata and pvmap files can be under statvar_imports/ndap/nss_health. There is no need for a pvmap/ folder.
If you are creating sub-folders, please use a sub-folder for an import and placed all files needed to process the import: places.csv, pvmap.csv and metadata.csv under that folder.